libraries/CAN/Makefile.test

```diff
@@ -2,12 +2,23 @@
 
 
 ### Example program
+## XMC4700_Relax_Kit
```

Comment:
How to run this? 
Where is the makefile? etc?

Reply:
I will keep this for now